src/auth/use-is-backend-sms-auth-enabled.tsx

```diff
@@ -4,12 +4,8 @@ import {StorageModelKeysEnum} from '@atb/storage';
 
 export const useIsBackendSmsAuthEnabled = () => {
   const {enable_backend_sms_auth} = useRemoteConfig();
-  const [debugOverride, _, debugOverrideReady] =
-  useBackendSmsAuthEnabledDebugOverride();
-  return [
-    debugOverride !== undefined ? debugOverride : enable_backend_sms_auth,
-    debugOverrideReady,
-  ];
+  const [debugOverride] = useBackendSmsAuthEnabledDebugOverride();
+  return debugOverride !== undefined ? debugOverride : enable_backend_sms_auth;
```

Comment:
Should we have something in Firestore also for the feature-toggle? If we at some point see there is something really wrong with the new login flow, we probably want to be able to toggle it instantly, not letting the user wait up to 6 hours?

Reply:
Talked about it in the office now, and we concluded that a 6 hour delay for this is fine. We have Vipps as alternative and global messages for info if needed.